src/modules/note.ts

```diff
@@ -228,47 +229,43 @@ const whois = new util.RootModule(
     },
   ],
   async (args, interaction) => {
-    const userArg: string = args
+    const targetId: string = args
       .find(arg => arg.name === 'user')!
       .value!.toString();
-    const member = await interaction.guild!.members.fetch(userArg).catch();
+    const member = await interaction.guild!.members.fetch(targetId).catch();
 
-    if (member === null) {
```

Comment:
I don't know how the semantics of `.catch()` work, this may be `undefined` instead of `null`. I have not validated this, but I would like to verify that you have.

Reply:
It shouldn't be possible period, since the user argument only permits members. I can try replicating it though

Reply:
This is actually  a relatively easy fix, pushing now